controls/automake_exists.rb

```diff
@@ -0,0 +1,23 @@
+title 'Tests to confirm automake exists'
+
+plan_origin = ENV['HAB_ORIGIN']
+plan_name = input('plan_name', value: 'automake')
+ 
+control 'core-plans-automake-exists' do
+  impact 1.0
+  title 'Ensure automake exists'
+  desc '
+  Verify automake by ensuring /bin/automake exists'
+  
+  plan_installation_directory = command("hab pkg path #{plan_origin}/#{plan_name}")
+  describe plan_installation_directory do
+    its('exit_status') { should eq 0 }
+    its('stdout') { should_not be_empty }
+  end
+
+  command_relative_path = input('command_relative_path', value: '/bin/automake')
```

Comment:
Can all inputs be grouped at the top of the file? @skpaterson correct me if I am wrong.

Reply:
Unnecessary

Reply:
Leaving this as discussed on slack.